Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -2688,6 +2693,17 @@ bool RenderLayerBacking::paintsContent(RenderLayer::PaintedContentRequest& reque
     if (request.isSatisfied())
         return paintsContent;
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    // FIXME: [LBSE] Eventually refine the logic to end up with a narrower set of conditions.
```

Comment:
Please file a bug to track this.

Reply:
Done. Added a link to webkit.org/b/243417.